ivy_tests/test_ivy/test_frontends/test_jax/test_jax_nn_activations.py

```diff
@@ -0,0 +1,42 @@
+import numpy as np
+from hypothesis import given, strategies as st
+
+
+# local
+import ivy_tests.test_ivy.helpers as helpers
+import ivy.functional.backends.jax as ivy_jax
+
+
+@given(
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=ivy_jax.valid_numeric_dtypes, min_value=-2
+    ),
+    as_variable=st.booleans(),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.jax.nn.softmax"
+    ),
+    native_array=st.booleans(),
+)
+def test_jax_nn_softmax(
+    dtype_and_x,
+    as_variable,
+    num_positional_args,
+    native_array,
+    fw,
+):
+    input_dtype, x = dtype_and_x
+    tmp = []
+    tmp.append(x)
+    data = np.asarray(tmp, dtype=input_dtype)
+
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        as_variable_flags=as_variable,
+        with_out=False,
+        num_positional_args=num_positional_args,
+        native_array_flags=native_array,
+        fw=fw,
+        frontend="jax",
+        fn_tree="nn.softmax",
+        x=data,
+    )
```

Comment:
We should be testing the axis parameter. We can generate this with `axis=helpers.ints(min_value=-1, max_value=0)` as done with the relevant backend test [here](https://github.com/unifyai/ivy/blob/971fba4acb10703b36b4d21aa5adeca3d09cc75d/ivy_tests/test_ivy/test_functional/test_nn/test_activations.py#L154)

Reply:
done, I have updated the test case